src/annotator/plugin/pdf.js

```diff
@@ -46,8 +46,8 @@ export default class PDF extends Delegator {
   /**
    * @param {Annotator} annotator
    */
-  constructor(element, config, annotator) {
-    super(element, config);
+  constructor(element, annotator) {
+    super(element);
```

Comment:
The `config` argument was unused, as was the `this.options` field under which `Delegator` exposes it.